joomla/joomla-cms
# [4.1] Simplify notification labels

### Summary of Changes
The legend states it is Notifications. Remove the redundant `Notifications on` to simplify labels.


### Testing Instructions
Go to System > Scheduled Tasks
Create a new task.
Go to Advanced tab.
See Notifications section.
